naptime-graphql/src/test/scala/org/coursera/naptime/ari/graphql/schema/ExecutorHelper.scala

```diff
@@ -0,0 +1,63 @@
+package org.coursera.naptime.ari.graphql.schema
+
+import com.linkedin.data.DataMap
+import org.coursera.naptime.ari.graphql.Models
+import org.coursera.naptime.ari.graphql.SangriaGraphQlContext
+import org.coursera.naptime.ari.graphql.SangriaGraphQlSchemaBuilder
+import org.coursera.naptime.ari.graphql.marshaller.NaptimeMarshaller._
+import org.coursera.naptime.ari.graphql.models.FakeModel
+import org.coursera.naptime.ari.graphql.models.MergedCourse
+import org.coursera.naptime.ari.graphql.models.MergedInstructor
+import org.coursera.naptime.ari.graphql.models.MergedPartner
+import org.coursera.naptime.ari.graphql.resolvers.NaptimeResolver
+import play.api.libs.json.JsObject
+import sangria.execution.Executor
+import sangria.parser.QueryParser
+import sangria.schema.Schema
+
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration
+
+class ExecutorHelper {
+
+  def executeQuery(
```

Comment:
this doesn't look super general, let's make the naming reflect that. Maybe `TestExecutorHelper` to start? sorry I don't have a better suggestion here :(